conda-forge.yml

```
@@ -1 +1,4 @@
+channels:
+  targets:
+  - [conda-forge, dev]
```

Indentation might be wrong:
```suggestion
    - [conda-forge, dev]
```

---

I guess we should setup dev label on the api client and migrate a working solution here when ready.